adr/queries/skipped_tests.query

```diff
@@ -0,0 +1,13 @@
+from: unittest
+select:
+    - {aggregate: count}
+groupby:
+    - {value: result.test}
+    - {value: run.suite}
+where:
+    and:
```

Comment:
Please add

    - eq {repo.branch.name: "mozilla-central"}

That should cut down the amount of data a fair bit.